src/deprecated/ChoiceFieldset/ChoiceFieldset.tsx

```diff
@@ -57,7 +57,7 @@ const {Slots, Slot} = createSlots(['Description', 'ChoiceList', 'Legend', 'Valid
 export {Slot}
 
 /**
- * @deprecated Use `CheckboxGroup` or `RadioGroup` instead.
+ * @deprecated Use `CheckboxGroup` or `RadioGroup` instead. See https://primer.style/react/CheckboxGroup and https://primer.style/react/RadioGroup for more info
```

Comment:
Similar feedback to the other annotation, I think this needs to be applied here:

```js
/**
  * @deprecated Use `CheckboxGroup` or `RadioGroup` instead. See https://primer.style/react/CheckboxGroup and https://primer.style/react/RadioGroup for more info
  */
export default Object.assign(ChoiceFieldset, {

```